GameProject/Assets/OpenWorld Assets/Scripts/ThirdPersonCharacterController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ThirdPersonCharacterController : MonoBehaviour
+{
+
+    [SerializeField] private float _moveSpeed;
+    [SerializeField] private float _rotateSpeed;
+    //private float rotX;
+    //private float rotY;
+    [SerializeField] private float _rotZ;
+    private Camera _cam;
+
+    private void Start()
+    {
+        _cam = Camera.main;
+    }
+
+    void Update()
+    {
+
+        PlayerTurn(); // turn to face the player
```

Comment:
These comments leave much to be desired. Please consider making the method names more concise instead.